homeassistant/components/upnp/sensor.py

```diff
@@ -176,7 +190,8 @@ def native_value(self) -> str | None:
 
         # Calculate derivative.
         delta_value = current_value - self._last_value
-        if self._sensor_type["unit"] == DATA_BYTES:
+        # pyright: reportGeneralTypeIssues=false
```

Comment:
Add `entity_description: UpnpSensorEntityDescription` as  a class attribute in this class and this line is no longer needed